manifests/server.pp

```diff
@@ -34,13 +34,17 @@
   Optional[Stdlib::Absolutepath] $krb5_keytab_file  = undef,
   Optional[String] $ldap_config_backend             = undef,
   Optional[Boolean] $enable_memory_limit            = undef,
-  Boolean $manage_policy_rc_d                       = true,
+  Optional[Boolean] $manage_policy_rc_d             = undef,
 ) {
   include openldap::server::install
   include openldap::server::config
   include openldap::server::service
   include openldap::server::slapdconf
 
+  unless $manage_policy_rc_d =~ Undef {
+    deprecation('manage_policy_rc_d', 'The manage_policy_rc_d parameter is deprecated and unused. It will be removed in a future version.')
```

Comment:
please, after this is merged, create a new PR that drops this and mark it as breaking change. otherwise we will forget about it.

Reply:
Yes! It's exactly what I want to do :grin:

Reply:
#388